### DISCUSSION TITLE
I am using UE5. How can I resolve PhysXVehicle plugin required.

### DISCUSSION THREAD

**Discussion by michael-2509** [microsoft/AirSim#4850] [Support Q&A] [answered]
I add the Airsim plugin to my unreal project and I run a build in visual studio 2022. I get the following errors below. How can I replace the PhysXVehicle plugin with ChaosVehicle Plugin? I've tried to manually enable the physXVehicle plugin in my .Uproject and yet it didn't work. 

![Screenshot (516)](https://github.com/microsoft/AirSim/assets/61965542/b16928e0-686f-48e2-b9bd-ef1068f2010b)
![Screenshot (517)](https://github.com/microsoft/AirSim/assets/61965542/850a0479-ff57-4e50-9172-8bde5ac0dc8e)
![Screenshot (518)](https://github.com/microsoft/AirSim/assets/61965542/76cc8aa5-ab3b-49b7-b2d1-826141f66a29)

Attempt 1
![image](https://github.com/microsoft/AirSim/assets/61965542/52323086-50ca-4235-8b10-babf083ea0cd)

ATttempt 2
![image](https://github.com/microsoft/AirSim/assets/61965542/e2ba9e1e-e05d-45b9-a688-7b8f6e450acc)

**Comment by xxEoD2242** [comment, marked as answer]
You can use [Colosseum](https://github.com/CodexLabsLLC/Colosseum)

**Reply by michael-2509** [reply]
Thank you

**Comment by F18810565662** [comment]
You can have a try with UE4. I made it. I think the reason may be that the project file was not associated with UE (which may require restarting the computer)
